src/main/java/de/zalando/aruha/nakadi/domain/EventType.java

```diff
@@ -32,8 +32,7 @@
     @JsonIgnore
     private final List<ValidationStrategyConfiguration> validationStrategies = Lists.newArrayList();
 
-    @Valid
-    private PartitionStrategyDescriptor partitionStrategy = RANDOM_PARTITION_STRATEGY;
+    private String partitionStrategy = PartitionStrategy.RANDOM_STRATEGY;
```

Comment:
I also don't really have a preference here. I just thought that String will allow us to be more flexible in the future.